miasm2/analysis/depgraph.py

```diff
@@ -110,7 +111,8 @@ def modifier(self):
     @modifier.setter
     def modifier(self, value):
         """Evaluating the current line involves a modification of tracked
-        dependencies"""
+        dependencies
+        @value: boolean"""
         if not isinstance(value, bool):
```

Comment:
We can consider that the docstring is enough and then the check is superfluous (in addition, this methods has speed concern).